src/platform/plugins/shared/discover/tsconfig.json

```diff
@@ -99,9 +98,8 @@
     "@kbn/core-lifecycle-browser",
     "@kbn/discover-contextual-components",
     "@kbn/esql-ast",
-    "@kbn/discover-shared-plugin"
+    "@kbn/discover-shared-plugin",
+    "@kbn/ui-theme"
```

Comment:
Not sure if this should have been removed completely, but looks like it was just moved down a few lines.

Reply:
if automation adds it back (check second commit) then I assume it should be there, because from the code I see ui-theme is still referenced. It should be cleaned up by the eui to borealis bigger epic eventually tho even tho it wasn't with this original PR